fonts/minecraft-ten/textures.json

```diff
@@ -213,7 +213,12 @@
       }
     },
       "bricks": {
-        "author": "IAmMaker"
+        "author": "IAmMaker",
+        "variants": {
+          "fancy_bricks":{
+            "name": "Fancy Bricks"
```

Comment:
This is not really needed as it will automatically create the name from the file

Reply:
ok